components/org.wso2.carbon.identity.api.server.branding.preference.management/org.wso2.carbon.identity.api.server.branding.preference.management.v1/src/main/resources/branding-preference.yaml

```diff
@@ -198,13 +198,190 @@ paths:
+                $ref: '#/components/schemas/CustomTextModel'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+    delete:
+      tags:
+        - Custom Text
+      operationId: deleteCustomText
+      summary: Deletes custom text.
+      description: |
+        This API provides the capability to delete the custom texts for the specified screen & locale of a tenant.<br>
+          <b>Permission required:</b> <br>
+            * /permission/admin/manage/identity/applicationmgt/update <br>
+          <b>Scope required:</b> <br>
+            * internal_application_mgt_update
+      parameters:
+        - $ref: '#/components/parameters/typeQueryParam'
+        - $ref: '#/components/parameters/nameQueryParam'
+        - $ref: '#/components/parameters/localeQueryParam'
+        - $ref: '#/components/parameters/screenQueryParam'
+      responses:
+        '204' :
+          description: Successfully deleted.
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+  '/branding-preference/text/resolve':
+    get:
+      tags:
+        - Custom Text
+      operationId: resolveCustomText
+      summary: Resolve custom text of an organization.
+      description: |
+        This API provides the capability to retrieve the custom text configurations of an organization/specific application.<br>
+        If there is no custom text available for the requested locale, API will check for the default locale('en-US') and return it.<br>
+        If there is no custom text available for the requested organization, API will check for the parent organization's custom text configurations and return it.<br>
+          <b>Permission required:</b> <br>
+            * None <br>
+          <b>Scope required:</b> <br>
+            * None
+      parameters:
+        - $ref: '#/components/parameters/typeQueryParam'
+        - $ref: '#/components/parameters/nameQueryParam'
+        - $ref: '#/components/parameters/localeQueryParam'
+        - $ref: '#/components/parameters/screenQueryParam'
+      responses:
+        '200':
+          description: OK
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/CustomTextModel'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
```

Comment:
not expected since this is an open endpoint

Reply:
Addressed with https://github.com/wso2/identity-api-server/pull/495/commits/1758c625c422ac94793b83d06c1f7732e44bc9d1